xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-notifications/src/main/resources/ApplicationResources.properties

```diff
@@ -25,14 +25,34 @@ mentions.mention.event.description=Mentions
 mentions.event.mention.type=Mentions
 mentions.event.mention.summary.singular=You have received one mention.
 mentions.event.mention.summary.plural=You have received {0} mentions.
-mentions.event.mention.description=mentioned you on page
-mentions.event.mention.description.document=mentioned you on page
-mentions.event.mention.description.comment=mentioned you on a comment on page
-mentions.event.mention.description.annotation=mentioned you on an annotation on page
-mentions.event.mention.description.awmField=mentioned you on a field on the AWM document
+mentions.event.mention.description.DOCUMENT=mentioned you on page {0}
+mentions.event.mention.description.COMMENT=mentioned you on a comment on page {0}
+mentions.event.mention.description.ANNOTATION=mentioned you on an annotation on page {0}
+mentions.event.mention.description.UNDEFINED=mentioned you on page {0}
+mentions.event.mention.description.TEXT_FIELD=mentioned you on a field on page {0}
 mentions.event.mention.title=New mention received
 
 ## Translations for default email notifications
 notifications.events.mentions.mention.description.by.1user=You have been mentioned by {0}.
 notifications.events.mentions.mention.description.by.users=You have received {0} mentions.
 notifications.events.mentions.mention.description=mentioned you on page
+
+###############################################################################
+## Deprecated
+## Note: each element should be removed when the last branch using it is no longer supported
+###############################################################################
+
+## Used to indicate where deprecated keys start
+#@deprecatedstart
+
+#######################################
+## until 14.10.7, 15.2RC1 
+#######################################
+mentions.event.mention.description=mentioned you on page {0}
```

Comment:
From what I see in the diff the previous translation did not have an argument: 
> mentions.event.mention.description=mentioned you on page

Reply:
done